src/Functions/URL/netloc.cpp

```diff
@@ -0,0 +1,17 @@
+#include <Functions/FunctionFactory.h>
```

Comment:
Please insert the name of this file to [Functions/ya.make](https://github.com/ClickHouse/ClickHouse/blob/master/src/Functions/ya.make) to fix Yandex synchronization check.